setup.cfg

```
@@ -61,6 +61,7 @@ cncf.kubernetes =
     kubernetes_asyncio
 databricks =
     apache-airflow-providers-databricks>=2.2.0
+    databricks-sql-connector>=2.0.4;python_version>='3.10'
```

Shouldnt we revert back this change back to 1.7.1 , now that we have 3.10 support?
```suggestion
    apache-airflow-providers-databricks>=2.2.0
```

---

Yes pip is able to look for compatible versions needed for python3.10 and installs the `databricks-sql-connector>=2.0.4`.
Will make the necessary changes.